data/locale/sv.ts

```diff
@@ -139,7 +139,7 @@ If you&apos;re interested in translating LMMS in another language or want to imp
     </message>
     <message>
         <source>If you enable this button, the whole sample is reversed. This is useful for cool effects, e.g. a reversed crash.</source>
-        <translation>Den här knappen gör att ljudfilen spelas baklänges. Den kan användas för intressanta effeker t.ex. en baklänges cymbal.</translation>
+        <translation>Om du aktiverar den här knappen så spelas hela ljudklippet upp baklänges. Det här är användbart för coola effeker, exempelvis vid användning av en baklänges-cymbal.</translation>
```

Comment:
I'm not sure "användbart" is a good translation. It's probably just me, but it sounds more like "usable" than "useful". Effeker -> Effekter (Once again my fault). I think "vid användning av" should be something different, because you "create" a backwards cymbal with the reverse button, you don't "use" the cymbal, you "use the button. "sorry for all the quotation marks" and if you'd prefer to discuss in swedish, just tell me. :)